### ISSUE TITLE
Crash: WordPressAuthenticator.swift L160

### ISSUE THREAD

**Issue by jleandroperez** [wordpress-mobile/WordPress-iOS#9210]
@frosty thanks for the stack trace!!!

```
Fatal error: WordPressAuthenticator wasn't initialized: file /Users/frosty/src/WordPress-iOS/WordPress/Classes/ViewRelated/NUX/WordPressAuthenticator.swift, line 160
2018-04-26 09:35:08.907187+0100 WordPress[1920:672286] Fatal error: WordPressAuthenticator wasn't initialized: file /Users/frosty/src/WordPress-iOS/WordPress/Classes/ViewRelated/NUX/WordPressAuthenticator.swift, line 160
(lldb) bt
* thread #1, queue = 'com.apple.main-thread', stop reason = EXC_BREAKPOINT (code=1, subcode=0x103858d9c)
    frame #0: 0x0000000103858d9c libswiftCore.dylib`function signature specialization <Arg[1] = Exploded> of Swift._assertionFailure(Swift.StaticString, Swift.String, file: Swift.StaticString, line: Swift.UInt, flags: Swift.UInt32) -> Swift.Never + 164
    frame #1: 0x00000001036f0130 libswiftCore.dylib`Swift._assertionFailure(Swift.StaticString, Swift.String, file: Swift.StaticString, line: Swift.UInt, flags: Swift.UInt32) -> Swift.Never + 40
    frame #2: 0x00000001008a74f4 WordPress`static WordPressAuthenticator.shared.getter(self=WordPress.WordPressAuthenticator) at WordPressAuthenticator.swift:160
    frame #3: 0x000000010079962c WordPress`static WordPressAuthenticator.track(event=openedLogin, self=WordPress.WordPressAuthenticator) at WordPressAuthenticator+Events.swift:12
    frame #4: 0x00000001008aa3cc WordPress`static WordPressAuthenticator.trackOpenedLogin(self=WordPress.WordPressAuthenticator) at WordPressAuthenticator.swift:295
    frame #5: 0x00000001008a89d4 WordPress`$defer #1 (self=WordPress.WordPressAuthenticator) in static WordPressAuthenticator.showLogin(from:animated:showCancel:restrictToWPCom:) at WordPressAuthenticator.swift:221
    frame #6: 0x00000001008a899c WordPress`static WordPressAuthenticator.showLogin(presenter=0x000000011489ce00, animated=true, showCancel=false, restrictToWPCom=false, self=WordPress.WordPressAuthenticator) at WordPressAuthenticator.swift:220
    frame #7: 0x00000001008a84c4 WordPress`static WordPressAuthenticator.showLoginFromPresenter(presenter=0x000000011489ce00, animated=true, self=WordPress.WordPressAuthenticator) at WordPressAuthenticator.swift:216
    frame #8: 0x00000001008a8554 WordPress`@objc static WordPressAuthenticator.showLoginFromPresenter(_:animated:) at WordPressAuthenticator.swift:0
    frame #9: 0x0000000100186094 WordPress`-[WordPressAppDelegate showWelcomeScreenAnimated:thenEditor:](self=0x00000001d42c02a0, _cmd="showWelcomeScreenAnimated:thenEditor:", animated=YES, thenEditor=NO) at WordPressAppDelegate.m:496
    frame #10: 0x0000000100185f0c WordPress`-[WordPressAppDelegate showWelcomeScreenIfNeededAnimated:](self=0x00000001d42c02a0, _cmd="showWelcomeScreenIfNeededAnimated:", animated=YES) at WordPressAppDelegate.m:490
  * frame #11: 0x000000010028abe0 WordPress`-[BlogListViewController maybeShowNUX](self=0x0000000113ff1cb0, _cmd="maybeShowNUX") at BlogListViewController.m:238
    frame #12: 0x000000010028a07c WordPress`-[BlogListViewController viewWillAppear:](self=0x0000000113ff1cb0, _cmd="viewWillAppear:", animated=NO) at BlogListViewController.m:167
    frame #13: 0x000000018d81e6d8 UIKit`-[UIViewController _setViewAppearState:isAnimating:] + 616
    frame #14: 0x000000018d81e448 UIKit`-[UIViewController __viewWillAppear:] + 140
    frame #15: 0x000000018d84c5b8 UIKit`-[UINavigationController _startTransition:fromViewController:toViewController:] + 784
    frame #16: 0x000000018d84bc90 UIKit`-[UINavigationController _startDeferredTransitionIfNeeded:] + 1164
    frame #17: 0x000000018dac5d48 UIKit`-[UINavigationController(StateRestoration) decodeRestorableStateWithCoder:] + 612
    frame #18: 0x000000018dc470d4 UIKit`-[UIViewController(StateRestoration) _decodeRestorableStateAndReturnContinuationWithCoder:] + 64
    frame #19: 0x000000018db89cc4 UIKit`___restoreState_block_invoke + 692
    frame #20: 0x000000018d7571c4 UIKit`+[UIView(Animation) performWithoutAnimation:] + 104
    frame #21: 0x000000018dac07c4 UIKit`-[UIApplication(StateRestoration) _restoreApplicationPreservationStateWithSessionIdentifier:beginHandler:completionHandler:] + 5512
    frame #22: 0x000000018d793204 UIKit`-[UIApplication(StateRestoration) _doRestorationIfNecessary] + 244
    frame #23: 0x000000018d792de4 UIKit`-[UIApplication _handleDelegateCallbacksWithOptions:isSuspended:restoreState:] + 324
    frame #24: 0x000000018d792240 UIKit`-[UIApplication _callInitializationDelegatesForMainScene:transitionContext:] + 3484
    frame #25: 0x000000018d75f65c UIKit`-[UIApplication _runWithMainScene:transitionContext:completion:] + 1680
    frame #26: 0x000000018dd8fa0c UIKit`__111-[__UICanvasLifecycleMonitor_Compatability _scheduleFirstCommitForScene:transition:firstActivation:completion:]_block_invoke + 784
    frame #27: 0x000000018d75ee4c UIKit`+[_UICanvas _enqueuePostSettingUpdateTransactionBlock:] + 160
    frame #28: 0x000000018d75ece8 UIKit`-[__UICanvasLifecycleMonitor_Compatability _scheduleFirstCommitForScene:transition:firstActivation:completion:] + 240
    frame #29: 0x000000018d75db78 UIKit`-[__UICanvasLifecycleMonitor_Compatability activateEventsOnly:withContext:completion:] + 724
    frame #30: 0x000000018e3f372c UIKit`__82-[_UIApplicationCanvas _transitionLifecycleStateWithTransitionContext:completion:]_block_invoke + 296
    frame #31: 0x000000018d75d268 UIKit`-[_UIApplicationCanvas _transitionLifecycleStateWithTransitionContext:completion:] + 432
    frame #32: 0x000000018e1d89b8 UIKit`__125-[_UICanvasLifecycleSettingsDiffAction performActionsForCanvas:withUpdatedScene:settingsDiff:fromSettings:transitionContext:]_block_invoke + 220
    frame #33: 0x000000018e326ae8 UIKit`_performActionsWithDelayForTransitionContext + 112
    frame #34: 0x000000018d75cc88 UIKit`-[_UICanvasLifecycleSettingsDiffAction performActionsForCanvas:withUpdatedScene:settingsDiff:fromSettings:transitionContext:] + 248
    frame #35: 0x000000018d75c624 UIKit`-[_UICanvas scene:didUpdateWithDiff:transitionContext:completion:] + 368
    frame #36: 0x000000018d75965c UIKit`-[UIApplication workspace:didCreateScene:withTransitionContext:completion:] + 540
    frame #37: 0x000000018d7593ac UIKit`-[UIApplicationSceneClientAgent scene:didInitializeWithEvent:completion:] + 364
    frame #38: 0x00000001863c0470 FrontBoardServices`-[FBSSceneImpl _didCreateWithTransitionContext:completion:] + 364
    frame #39: 0x00000001863c8d6c FrontBoardServices`__56-[FBSWorkspace client:handleCreateScene:withCompletion:]_block_invoke_2 + 224
    frame #40: 0x00000001041ed220 libdispatch.dylib`_dispatch_client_callout + 16
    frame #41: 0x00000001041f9850 libdispatch.dylib`_dispatch_block_invoke_direct + 232
    frame #42: 0x00000001863f4878 FrontBoardServices`__FBSSERIALQUEUE_IS_CALLING_OUT_TO_A_BLOCK__ + 36
    frame #43: 0x00000001863f451c FrontBoardServices`-[FBSSerialQueue _performNext] + 404
    frame #44: 0x00000001863f4ab8 FrontBoardServices`-[FBSSerialQueue _performNextFromRunLoopSource] + 56
    frame #45: 0x0000000183b6b404 CoreFoundation`__CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE0_PERFORM_FUNCTION__ + 24
    frame #46: 0x0000000183b6ac2c CoreFoundation`__CFRunLoopDoSources0 + 276
    frame #47: 0x0000000183b6879c CoreFoundation`__CFRunLoopRun + 1204
    frame #48: 0x0000000183a88da8 CoreFoundation`CFRunLoopRunSpecific + 552
    frame #49: 0x0000000185a6b020 GraphicsServices`GSEventRunModal + 100
    frame #50: 0x000000018da6978c UIKit`UIApplicationMain + 236
    frame #51: 0x000000010017d990 WordPress`main(argc=1, argv=0x000000016fc8b9e0) at main.m:11
    frame #52: 0x0000000183519fc0 libdyld.dylib`start + 4
```
```

**Comment by jleandroperez** [comment]
Updating the Milestone to WPiOS 10.0, since 9.9 is safe (can't repro the crash there)!

**Comment by diegoreymendez** [comment]
Removed milestone since this didn't make the 9.9 cut.

**Comment by jleandroperez** [comment]
Closed via #9212
